From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Wed, 23 Nov 2022 21:03:04 +0100
Subject: [PATCH] Fiddle configuration

License: MIT (https://opensource.org/licenses/MIT)
Fiddle - https://fiddlemc.org

This patch is based on the following patch:
"Gale configuration"
By: Martijn Muijsers <martijnmuijsers@live.nl>
As part of: Gale (https://github.com/GaleMC/Gale)
Licensed under: MIT (https://opensource.org/licenses/MIT)

* Gale description *

License: MIT (https://opensource.org/licenses/MIT)
Gale - https://galemc.org

This patch is based on the following patch:
"Timings v2"
By: Aikar <aikar@aikar.co>
As part of: Paper (https://github.com/PaperMC/Paper)
Licensed under: MIT (https://opensource.org/licenses/MIT)

diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index 8621dba7f10ab822b5b99ce0d05da58823605cb6..ec5ab7d0cb08b9c87dd70296a3a615c734fb54c7 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -1966,6 +1966,15 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         {
             throw new UnsupportedOperationException("Not supported yet.");
         }
+
+        // Fiddle start - Fiddle configuration - API
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getFiddleConfig()
+        {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // Fiddle end - Fiddle configuration - API
+
         // Paper end
 
         /**
